geopandas/tests/test_geocode.py

```diff
@@ -13,16 +14,14 @@
 from pandas.util.testing import assert_series_equal
 from geopandas.tests.util import mock, assert_geoseries_equal
 
+geopy = pytest.importorskip("geopy")
 
-def _skip_if_no_geopy():
-    try:
-        import geopy
-    except ImportError:
-        raise pytest.skip("Geopy not installed. Skipping tests.")
-    except SyntaxError:
-        raise pytest.skip("Geopy is known to be broken on Python 3.2. "
-                          "Skipping tests.")
+from geopy.exc import GeocoderNotFound
+from geopy.geocoders import GoogleV3
```

Comment:
I am not sure this will work (those lines will still be executed, I assume). It might be needed to put those imports inside the test functions.

Reply:
Fixed.